### ISSUE TITLE
Background color doesn't work

### ISSUE THREAD

**Issue by gasci** [Cap-go/capacitor-inappbrowser#138]
The backgroundColor parameter doesn't change anything on iOS. It is always white.

**Comment by riderx** [comment]
did you tried with `openWebView` ? that the only one who work and it should be only black or white

**Comment by gasci** [comment]
I see the section safe-area-bottom always white on iOS. Maybe we need another parameter for that?

**Comment by riderx** [comment]
Can you share a screenshot?

**Comment by gasci** [comment]
```
const loading = await showLoading({ message: undefined });
await InAppBrowser.openWebView({
  url: modifiedUrl,
  toolbarType: ToolBarType.NAVIGATION,
  showReloadButton: reloadButton,
  isPresentAfterPageLoad: true,
  activeNativeNavigationForWebview: true,
  title: title,
  showArrow: false,
  backgroundColor: BackgroundColor.BLACK
}).finally(() => {
  dismissLoading(loading)
})
```

Here are my screen shots:
![Screenshot 2024-06-14 at 19 55 19](https://github.com/Cap-go/capacitor-inappbrowser/assets/29505701/246917e6-ce7c-40b1-aacd-e5c0e33c52ae)

**Comment by riderx** [comment]
I don't understand the issue on your screenshot, the top bar is white it's normal the bottom is white

**Comment by gasci** [comment]
Yes, the backgroundColor and toolbarColor parameters have been completely ineffective somehow on iOS 17.5.

**Comment by WcaleNieWolny** [comment]
Hey, I believe that I found your issue.

For reference, I set the `backgroundColor` to `red`
![image](https://github.com/user-attachments/assets/aa4ca9f1-c3bb-42e9-9f74-eb38a866623a)

Right now, the backgroundColor really doesn't work, but it also influences the toolbar:

![image](https://github.com/user-attachments/assets/77044e80-8194-479b-b51a-594bd5a95b2d)

Furthermore, the toolbar colors [don't work as well](https://github.com/Cap-go/capacitor-inappbrowser/issues/198)

**Comment by gasci** [comment]
Yes, I had stopped using those parameters. :)

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by riderx** [comment]
fixed i redid this from the ground in 7.5
